### ISSUE TITLE
Not able to import MIF-files

### ISSUE THREAD

**Issue by summamutikka** [oskariorg/oskari-docs#229]
Importing MIF files fails. Tested with Paikkatietoikkuna and demo.oskari.org. 

Test data was downloaded from https://tiedostopalvelu.maanmittauslaitos.fi/tp/kartta. This data opens fine in Mapinfo Pro. Tested also with data exported as MIF from QGIS & Mapinfo Pro and all fail to import to geoportal. 

![image](https://user-images.githubusercontent.com/31657333/94234233-fa794580-ff11-11ea-951a-1f32184d2571.png)

**Comment by jampukka** [comment]
Seems like it the issue is with 
```
DATA
PLINE 53
508688.045709824 7722000
508639.620000001 7722170.699
508607.842000001 7722398.581
...
```
the parser assumes `PLINE` as
```
PLINE [ MULTIPLE numsections ] 
numpts1
x1 y1
x2 y2
```
but here `numpts1` appears on the same line as `PLINE`.
I don't have access to version 900 of MIF spec to check if this is allowed, but we could add some leniency to the parser to make it work with such files.

**Comment by jampukka** [comment]
The function that handles this is:
https://github.com/oskariorg/oskari-server/blob/master/geotools-ext/gt-mif/src/main/java/org/geotools/mif/MIFDataReader.java#L170

I guess the logic would be something like:
if `MULTIPLE` follows `PLINE` then get `numsections` from third item, otherwise check if second item on PLINE is an integer then use that as numpts1. If there's no values on the PLINE line then assume numpts1 will appear on the next line.

**Comment by jampukka** [comment]
@ZakarFin feel free to Assign this to me.

**Comment by jampukka** [comment]
Please check if https://github.com/oskariorg/oskari-server/pull/649 solves the issue for you

**Comment by ZakarFin** [comment]
oskariorg/oskari-server#649 is a partial fix for this but doesn't solve it completely. Here's a mid/mif zip that I used for testing:
[data.zip](https://github.com/oskariorg/oskari-docs/files/5346581/data.zip)

**Comment by jampukka** [comment]
Thanks I'll check it later this week.

**Comment by jampukka** [comment]
The issue was a bit more wider than anticipated affecting not only `LINE`s but also `POINTs`, `ARC`s, and `MULTIPOINT`s. The fix however was tiny. Please check if https://github.com/oskariorg/oskari-server/pull/653 solves the issue.

**Comment by ZakarFin** [comment]
It works now with the zip I used for testing so resolving this. The fix will be included in Oskari 2.1.0. Thanks @jampukka!
